docs/README.md

```diff
@@ -1,355 +1,167 @@
-This will generate local api docs in the `_api-extractor-temp` directory.
+The second option, which is substantially slower, leverages the same build that our build pipelines use to generate our production site.
+First, run:
 
-```bash
+```shell
 npm run build
 ```
 
-Once the local api docs is generated, run the `build:local` script.
-The output will be in the `public/` folder.
-
-```bash
-npm run build:local
-```
-
-Note that this calls the `local-api-rollup` script which simulates the `download` process in the regular build.
-However, instead of downloading the api content, the `_doc-models` directory is populated by copying the models from `_api-extractor-temp`.
-The local api content from `_api-extractor-temp` is copied to `_doc-models/local`.
-
-
-### Drafts
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+This includes the generation of API documentation contents.
 
-Work-in-progress documents that are not ready for public consumption can be safely added by annotating them with the `draft` flag in their frontmatter.
+To _just_ run content generation steps, run `build:generate-content`.
+To _just_ build the static site (without rebuilding the API documentation), run `build:site`.
 
-Example:
+Then, run:
 
-```markdown
----
-title: "Foo"
-draft: true
----
+```shell
+npm run serve
 ```
 
-Such documents will be disregarded by the build (by default) and will not be published.
+Note: the Docusaurus build is fairly slow.
+If you don't need to test search, it is recommended to run `npm start` instead.
+This is faster, and will watch for content changes and update automatically.
+You will still need to build the API documentation first.
 
-Drafts are a good option for making incremental progress on a document via pull requests before being ready to actually publish for the world to see.
+### Local API docs build
 
-For more documentation on this Hugo feature, see [here](https://gohugo.io/getting-started/usage/#draft-future-and-expired-content).
+To include repo-local API documentation when building the site locally, you will first need to do the following:
 
-#### Previewing drafts locally
+1. Build the code from the root of the repo.
+2. Create a `.env` file in this directory with the following contents:
+    ```
+    LOCAL_API_DOCS=true
+    ```
 
-As noted above, the `build` script won't build `draft` content.
-To build this content and preview it locally, you can run the build with the `--buildDrafts` flag.
+So long as the `LOCAL_API_DOCS` environment variable is set to `true`, local API documentation will be included when building the site.
+To remove the local API docs, simply remove the above variable or set it to `false`, `npm run clean` and rebuild as needed.
 
-```bash
-npm run build -- --buildDrafts
-```
+## Writing site documentation
 
-### Future Content
+TODO
```

Comment:
TODO? Also below in "Leveraging React components".

Reply:
I've filled in some of this now, and filed tasks to come back and add more details.